public/less/bootstrap.less

```diff
@@ -50,3 +50,7 @@
 .panel {
   border: none;
 }
+
+.staging textarea.form-control& {
```

Comment:
Can you please move that into [`staging.less`](https://github.com/FredrikNoren/ungit/blob/1b6ccefab995893f3cc3a353e976217a258c867a/components/staging/staging.less)?

Reply:
No worries, sorry -- don't know how I missed that.